### ISSUE TITLE
Provide assets for the left and right hand

### ISSUE THREAD

**Issue by cabanier** [immersive-web/webxr-input-profiles#178]
There currently is no standard model for the user's hands.
We should provide one, even if the UA doesn't support the joints API.

@thetuvix @RafaelCintron, does Microsoft have a GLB model that supports all the joints?

**Comment by cabanier** [comment]
/agenda discuss where we can get the hand model from. Also, should we standardize the name of the hands? (ie on quest, the UA reports `Oculus hand`)

**Comment by Manishearth** [comment]
I don't know what you mean by the name of the hands?

We have a generic-hands-select profile, y'all are free to create an oculus-hands-select profile too (and upload assets that are most like the native oculus ones if that ends up being different from generic)

But ideally we can get a hand model for the generic one first.

**Comment by cabanier** [comment]
I'm unsure if we should have different hands for different platforms. Shouldn't we use the same hand?

**Comment by Manishearth** [comment]
I think we should have a single generic hand model, but if vendors wish to add their own they can. I don't feel strongly about allowing or disallowing the latter.

For now I think we can just stick to generic-hand-select and add a model

**Comment by thetuvix** [comment]
There are two key differences here between hand models vs. controller models:
* Controller models are hierarchies of articulating rigid meshes, while even stylized hand models will need to be skinned.
* Unlike different per-profile (per-vendor/per-product) controller models, the same human hands will be using XR devices across all vendors.

It's great for us to publish libraries that make it turnkey to render hands based on WebXR data.  I'm not sure if it specifically fits in the webxr-input-profiles repo, as I don't see any per-profile hand rendering differences here.  However, if we had such a hand rendering library in some neighboring repo, I could see us generalizing the [motion-controllers module](https://github.com/immersive-web/webxr-input-profiles/blob/master/packages/motion-controllers/README.md) to pull that in too when any hand profile is in use.

**Comment by Manishearth** [comment]
> However, if we had such a hand rendering library (in some neighboring repo or perhaps just in another package here if it's simpler), I could see us generalizing the [motion-controllers package](https://github.com/immersive-web/webxr-input-profiles/blob/master/packages/motion-controllers/README.md) to pull that in too when any hand profile is in use.

Yeah, sorry, when I said input profiles, I meant the motion-controllers package. We _could_ support per-hand-profile models (e.g. so that Oculus devices get Oculus hand models, etc), but I don't think this is super necessary.
